app/controllers/groups_controller.rb

```diff
@@ -34,6 +34,9 @@ def create
       @group.add_admin! current_user
       @group.creator = current_user
       Measurement.increment('groups.create.success')
+      if @group.is_parent?
```

Comment:
Not a show stopper but more an FYI.. this should be moved into the GroupService and out of the controller. I'm sure it will be moved real soon as we enable group creation in the angular interface